app/views/admin/books/_new_book.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_for :book do |f| %>
+  <div class="field">
+    <%= f.label :title, t(".new_book") %>
+    <%= f.text_field :title, class: "form-control",
+      placeholder: t(".enter_name") %>
+
+    <%= f.label :num_page, t(".num_page") %>
+    <%= f.text_field :num_page, class: "form-control",
+      placeholder: t(".enter_name") %>
+
+    <%= f.label :description, t(".description") %>
+    <%= f.text_area :description, placeholder: t(".enter_name") %>
+
+    <%= f.label :author_id, t(".author") %>
+    <%= f.collection_select :author_id, Author.order(:name),:id,:name,
+      include_blank: false %>
+
+    <%= f.label :category_id, t(".category") %>
+    <%= f.collection_select :category_id, Category.order(:name),:id,:name,
```

Comment:
1. Perform this operation `Category.order(:name)` in controller 
2. Add necessary white space